nafis-vw-header/includes/css/scss/_module-header-brand-dealership.scss

```
@@ -0,0 +1,81 @@
+.brandDealership {
+	display: flex;
+	align-items: center;
+	position: relative;
+
+	&::before,
+	&::after {
+		content: "";
+		height: 2px;
+		width: 100%;
+		background-color: $white;
+		transition: background-color $transition-duration ease-in;
+
+		.headerInformation:hover & {
+			background-color: $black;
+		}
+	}
+}
+
+.logoDealerNameWrapper {
+	display: flex;
+	align-items: center;
+	gap: 20px;
+	min-width: 100vw;
+
+	@include respond-screen($bp-lg-2) {
+		min-width: $container-width;
+	}
+
+	&::before {
+		content: "";
+		height: 2px;
+		width: 5%;
+		background-color: $white;
+		transition: background-color $transition-duration ease-in;
+	}
+
+	&::after {
+		content: "";
+		height: 2px;
+		flex-grow: 1;
+		background-color: $white;
+		transition: background-color $transition-duration ease-in;
+	}
+
+	.headerInformation:hover & {
+		&::before,
+		&::after {
+			background-color: $black;
+		}
+	}
+}
+
+.logoDealerNameLink {
+	display: flex;
+	align-items: center;
+	gap: 20px;
+	text-decoration: none;
+}
+
+.brandLogo {
+	width: $brand-logo-size;
+	height: $brand-logo-size;
+	fill: $white;
+	transition: fill $transition-duration ease-in;
+
+	.headerInformation:hover & {
+		fill: $black;
+	}
+}
+
+.dealerName {
```

Refactor class name to indicate it's text
```suggestion
.dealerNameText {
```